### DISCUSSION TITLE
If I self hosted RocketChat, is it totally free and full function?

### DISCUSSION THREAD

**Discussion by thanhpd56** [RocketChat/Rocket.Chat#30144] [General]
I see here https://www.rocket.chat/pricing there are many things different between community and enterprise version.
For example some Omnichannels are not supported
<img width="995" alt="image" src="https://github.com/RocketChat/Rocket.Chat/assets/3794898/60c90f59-183f-4982-bd72-44f5111d15fd">

**Comment by sandrotanner** [comment]
No sadly not. Besides the features you found they also heavily limit the amount of push notifications you can send through their gateway (while simultaneously making it hard to run your own)

**Reply by reetp** [reply]
It is what it is.

Rocket.Chat have to feed their devs somehow.

Push notifications - they don't make it 'difficult' for you to run your own. It is just not easy to do it as you need to rebuild apps etc.

Starter is pretty well fully functional and free for 50 users. If you have more than 50 and you are a community project etc then talk to sales.

Who do you think should pay?

If you are a business with more that 50 employees you must be earning some profit somewhere..... You can build as CE only, or you can use some of your expenses to help fund future development and support for Rocket.Chat. 

Remember - If they go bust you have no code at all.

**Comment by Samira38** [comment]
Can somebody help me?
I installed Rocket chat for my workplace as a standalone server. Everything has worked perfectly. But I can not receive any notification on my Mobile phone. I surfed on the Internet, and I found out that in order to receive message notification I must register my server in Rocket chat cloud.  It is necessary to do that?

**Reply by reetp** [reply]
In a word, yes.

Otherwise you have to build your own open source model.

**Comment by arjun-zosma** [comment]
If I am self hosting it, why does it say free upto 50 users? Why should I even self host it?

Pure non sense and wasted my time doing all the installation and integration upto some extent.

**Reply by reetp** [reply]
> If I am self hosting it, why does it say free upto 50 users? 

Because it is, but you have to register to get service such as push notifications, apps etc.

> Pure non sense and wasted my time doing all the installation and integration upto some extent.

It's in the docs if you read them.

**Comment by reetp** [comment]
The correct places to ask these questions are here:

https://open.rocket.chat/channel/support/
https://forums.rocket.chat/
